gen.sh

```diff
@@ -0,0 +1,2 @@
+#cargo run --manifest-path=frb_codegen/Cargo.toml -- --rust-input frb_example/with_flutter/rust/src/api.rs --dart-output frb_example/with_flutter/lib/bridge_generated.dart
+cargo run --manifest-path=frb_codegen/Cargo.toml -- --rust-input something/src/api.rs --dart-output something/bridge_generated.dart
```

Comment:
Please use `justfile` instead of creating a new .sh

Reply:
not going to be in the final PR